http/src/main/resources/reference.conf

```diff
@@ -431,7 +431,7 @@ image {
   # Note: Since this is a GCR image crypto detection is currently disabled for AoU which requires
   # non-GCR images. To enable on AoU we would need to replicate the image in quay.io or potentially
   # ACR (Broad is deprecating use of Dockerhub).
-  cryptoDetectorImage = "us.gcr.io/broad-dsp-gcr-public/cryptomining-detector:0.0.1"
+  cryptoDetectorImage = "us.gcr.io/broad-dsp-gcr-public/cryptomining-detector:rt-test"
```

Comment:
TODO update to 0.0.2 when https://github.com/broadinstitute/terra-cryptomining-security-alerts/pull/3/ is merge/pushed.